sdk/highlight-go/log/logrus.go

```diff
@@ -78,22 +77,8 @@ func (hook *Hook) Fire(entry *logrus.Entry) error {
 		}
 	}
 
-	hasError := false
 	for k, v := range entry.Data {
-		if k == "error" {
-			hasError = true
-			if err, ok := v.(highlight.ErrorWithStack); ok {
-				highlight.RecordSpanErrorWithStack(span, err)
-			} else if err, ok := v.(error); ok {
-				span.RecordError(err, trace.WithStackTrace(true))
-			}
-		} else {
-			attrs = append(attrs, attribute.String(k, fmt.Sprintf("%+v", v)))
-		}
-	}
```

Comment:
this code meant that any `logrus.WithError(err)....` would report a highlight error object, even if the log was an `info`. Changing this since we don't want logs to ever be reported as an error (we just want that `WithError` to be a structured log attribute),